sdks/go/pkg/beam/core/runtime/exec/fn_test.go

```diff
@@ -708,11 +708,11 @@ func (c *callerInt) Call1x1(a0 interface{}) interface{} {
 // * Implicit or Explicit method Receiver
 // * Pre-wrapped values and pre-allocated slices.
 // * Invocations along the following ways
-//   * Indirect via extracting from a reflect.Value.Interface()
-//   * Reflect Package (reflect.Value.Call())
-//   * Beam's reflectx.Func, and reflectx.FuncNxM interfaces
-//       * Beam's default reflection based reflectx.Func shim
-//       * A Type assertion specialized reflectx.Func shim
+//   - Indirect via extracting from a reflect.Value.Interface()
+//   - Reflect Package (reflect.Value.Call())
+//   - Beam's reflectx.Func, and reflectx.FuncNxM interfaces
+//   - Beam's default reflection based reflectx.Func shim
+//   - A Type assertion specialized reflectx.Func shim
```

Comment:
Technically the sub indents were describing reflectx.Func and reflectx.FuncNxM respectively, so it was intending to serve a purpose.  In this case, I agree that there's no reason to fuss over it since it was already broken.